### DISCUSSION TITLE
Merge two Github Organization account

### DISCUSSION THREAD

**Discussion by KoushikCU** [community/community#49978] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi All,

I've created a Github Organization account and activated the Coupon code. however, now i notice that there was already an existing Organization account from our company for which the coupon suppose to be activated.

Is there any possibility to merge/ transfer the account to our company account? please suggest.

Regards,
Koushik
SkyPoint Cloud

**Comment by neutrinomuon** [comment]
Hello!!!

Yes, it is possible to transfer ownership of one GitHub organization to another organization. Here are the steps to follow:

- Ensure that you have admin access to both the organizations involved in the transfer.
- Navigate to the organization that you want to transfer.
- Click on "Settings" and then click on "Transfer ownership".
- Follow the prompts to enter the name of the organization you want to transfer the ownership to, and then click on "Transfer".

The organization's ownership will be transferred to the new organization.

**IMPORTANT:** Once the transfer is complete, the repositories, issues, pull requests, and other organization data will be transferred to the new organization. It is important to note that this action cannot be undone, so _make sure that you have selected the correct organization before proceeding with the transfer._

Also, make sure to review GitHub's documentation on transferring ownership of organizations to ensure you understand the process fully and that you are complying with all relevant policies and guidelines.

I hope this helps :)!

**Reply by Jdban** [reply]
Is this still possible? I don't see it in the UI anywhere

**Reply by neutrinomuon** [reply]
Maybe not... I haven't tried it recently. But there was an option to transfer ownership before. If I have some time today I'll check it and come back here

**Reply by neutrinomuon** [reply]
@Jdban If you go to the repository - Settings then you find the option Transfer ownership (see the screenshot below) to other users or organizations:

![window](https://github.com/community/community/assets/8226984/509f559c-07f9-404f-9449-b4e53075d55b)

So, it is still possible, at least, to transfer the repositories.

**Reply by neutrinomuon** [reply]
From a quick inspection, you're right. I see no clear documentation. What I've found is this document concerning the transfer of the organization's ownership. Maybe it can also help:

https://docs.github.com/en/organizations/managing-organization-settings/transferring-organization-ownership

**Reply by Jdban** [reply]
I see the option to transfer repositories, but not merge a whole organization

![image](https://github.com/community/community/assets/490005/10d2db64-ed68-4a90-afed-baa4f3498646)

My company has a couple organizations that need to be merged into one, and we're trying to figure out the best way to do that.

**Reply by garbelini** [reply]
@Jdban Have you found out anything new about this?

**Reply by Jdban** [reply]
> @Jdban Have you found out anything new about this?

It doesn't seem like it's possible to merge in an automated way (unless you write some automation yourself maybe)

**Reply by garbelini** [reply]
> It doesn't seem like it's possible to merge in an automated way (unless you write some automation yourself maybe)

Gotcha. Thanks for validating.
